packages/react/src/components/form/initial.tsx

```diff
@@ -113,12 +118,16 @@ const HandleForm: React.FC<HandleFormProps> = ({
     (f) => !isFactorOidc(f)
   );
   const shouldRenderFactorDropdown = filteredFactors.length > 1;
-  const [email, setEmail] = useState("");
-  const [phone, setPhone] = useState("");
+  const { registerField, registerSubmit, values, status, resetForm } =
+    useForm();
   const [flag, setFlag] = useState<Flag>(GB_FLAG);
   const [factor, setFactor] = useState<Factor>(filteredFactors[0]);
   const { text } = useConfiguration();
 
+  useEffect(() => {
+    return resetForm;
```

Comment:
cleanup when changing tabs so we don't have stale errors

Reply:
Nice!